packages/puppeteer-core/package.json

```diff
@@ -75,9 +74,6 @@
         "src/generated/**"
       ]
     },
-    "clean:third_party": {
```

Comment:
This was needed for incremental builds to work. Could you test the incremental build?

The problem is that we first roll-up and then tsc, on the second build the tsc'ed classes are used in the compilation without re-running rollup. And that led to the compilation errors on the second run of the build. 

The better solution would be to tsc into a temp dir and then rollup back into the target dir. Do you think that could work?

Reply:
Hmm, I'm not sure what you mean by this. Incremental builds are supported within `wireit` using the `clean: if-file-deleted` option according to their documentation.

Reply:
I verified it in this branch, the problem is not there anymore. Perhaps because we don't try to bundle bidi anymore. I can explain it over a VC if you want to understand the problem. I think the definition of build:third_party is not correct because the output folder is the input folder at the same time.

Reply:
Ok, I see what you mean. Let me try something.